zerver/webhooks/opencollective/doc.md

```diff
@@ -1,19 +1,25 @@
-This integration currently supports getting notifications to a channel of your Zulip instance,
-when a new member signs-up on an **Open Collective** page.
+# Zulip Open Collective integration
+
+Get Zulip notifications for new sign-ups on your **Open Collective** page.
+
+{start_tabs}
 
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to [Open Collective Website](https://opencollective.com/), find
-your desired collective page, then go to *Settings* -> *Webhooks*, paste the
-bot URL and choose *Activity* -> *New Member*.
+your desired collective page, then go to **Settings** > **Webhooks**, paste the
+generated URL, and choose **Activity** > **New Member**.
+
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/opencollective/001.png)
 
-In the future, this integration can be developed in order to
-support other types of *Activity* such as *New Transaction*, *Subscription Canceled* etc.
```

Comment:
Can this be added as a note?

Reply:
I agree with Pieter that we can remove this from this doc, as the docs are aimed at users. But to retain the point, we could add it to some other platform where follow-ups are tracked?

Reply:
Removing this from the documentation seems fine.